src/Console/ServeCommand.php

```diff
@@ -0,0 +1,40 @@
+<?php namespace TightenCo\Jigsaw\Console;
+
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class ServeCommand extends Command
+{
+    public function __construct()
+    {
+        parent::__construct();
+    }
+
+    protected function configure()
+    {
+        $this->setName('serve')
+            ->setDescription('Serve local site with php built-in server.')
+            ->addArgument(
```

Comment:
This is nice, but there's a subtle difference between an Optional Argument, and an Option. With arguments, the order matters. With options you have a `--option=...` prefix (or you can define a single letter shortcut like `-o=...`.

I think you want instead to 

```php
->addOption(
    'environment',
    'e',
    InputOption::VALUE_REQUIRED,
    'What environment should be served?'
)
```
Port should be an option as well. 

Then you can call it like so:
`$ jigsaw serve -e=production -p=8080` or `$ jigsaw serve --port=8080` (note that the order no longer matters, whereas this would if they were arguments because arguments don't have prefixes.)